### PR TITLE
[MRG] support `mle report <org/repo>` command

### PR DESCRIPTION
**Pull request by leeeizhang**
closes: https://github.com/MLSysOps/MLE-agent/issues/164

### checklists
- [x] support `mle report` cli
- [ ] ~~add fetch github metadata api~~ (move to another PR)

```
$ mle report MLSysOps/MLE-agent
```
image

### CODE REVIEW

File: mle/cli.py [hunk 1]
```diff
@@ -59,12 +60,40 @@ def start(mode, model):
...
+        console.log("Invalid github repository, "
+                    "Usage: 'mle report <organization/name>'")
+        return False
+
+    if not check_config():
+        # build a new project for github report generating
+        project_name = f"mle-report-{repo.replace('/', '_').lower()}"
+        ctx.invoke(new, name=project_name)
+        # enter the new project for report generation
+        work_dir = os.path.join(os.getcwd(), project_name)
```
Comment (huangyz0918) [inline mle/cli.py:92]: the project will be created at the current command execution path?

Comment (leeeizhang) [reply mle/cli.py:92]: yes, is it unreasonable for users? or we may enforce the user to new a project, before calling `mle report`.

Comment (huangyz0918) [reply mle/cli.py:92]: no, it is ok to me : ) Just want to clarify

**Review by huangyz0918 — APPROVED** [review]
LGTM

**Comment by HuaizhengZhang** [comment]
@leeeizhang Please update readme accordingly

**Pull request merged**